### ISSUE TITLE
Appearance compatibility? 

### ISSUE THREAD

**Issue by adou600** [raulriera/TextFieldEffects#30]
Hi,

I'm trying to use appearance to customize an HoshiTextField instance. 

In the AppDelegate, I call the Appearance method:

```
@UIApplicationMain
class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDelegate {

    var window: UIWindow?


    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
        
        Appearance().applyStyling()
        ...
```

Which contains:

```
class Appearance: NSObject {

    func applyStyling() {
        HoshiTextField.appearance().backgroundColor = UIColor.redColor() //will work
        HoshiTextField.appearance().placeholderColor = UIColor.redColor() // will fail 
    }
}
```

Doing that, I get a EXC_BAD_ACCESS in the second line of the method. Any idea why this isn't working? Is it somehow not compatible with appearance yet?

Thanks for your help and for your really cool plugin!

**Comment by raulriera** [comment]
Hmmm I don't have much experience enabling custom Appearance proxy support. I will take a look and see what I can do.

Thanks!

**Comment by raulriera** [comment]
Fixed in the 0.4.3 release

Thanks for reporting this!

**Comment by adou600** [comment]
Thanks, it works like a charm now! 
Fast and reliable work, cool! Thanks!
